### ISSUE TITLE
R 3.6.3 fails to install on SLES 12.5

### ISSUE THREAD

**Issue by joncfoo** [rstudio/r-builds#82]
<details>
<summary>Logs</summary>

```
 Downloading installer from https://cdn.rstudio.com/r/opensuse-42/pkgs/R-3.6.3-1-1.x86_64.rpm...

Downloading https://cdn.rstudio.com/r/opensuse-42/pkgs/R-3.6.3-1-1.x86_64.rpm...

User install from RPM installer R-3.6.3-1-1.x86_64.rpm...

Adding repository 'Backport builds of Python Modules (SLE_12_SP5)' [......done]

Repository 'Backport builds of Python Modules (SLE_12_SP5)' successfully added

URI : https://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_12_SP5/

Enabled : Yes

GPG Check : Yes

Autorefresh : No

Priority : 99 (default priority)

Repository priorities are without effect. All enabled repositories share the same priority.

Updating package indexes...

Repository 'SLE-Module-Adv-Systems-Management12-Pool' is up to date.

Repository 'SLE-Module-Adv-Systems-Management12-Updates' is up to date.

Repository 'SLE-Module-Containers12-Pool' is up to date.

Repository 'SLE-Module-Containers12-Updates' is up to date.

Repository 'SLE-Module-HPC12-Pool' is up to date.

Repository 'SLE-Module-HPC12-Updates' is up to date.

Repository 'SLE-Module-Legacy12-Pool' is up to date.

Repository 'SLE-Module-Legacy12-Updates' is up to date.

Repository 'SLE-Module-Public-Cloud12-Pool' is up to date.

Repository 'SLE-Module-Public-Cloud12-Updates' is up to date.

Repository 'SLES12-SP5-Pool' is up to date.

Repository 'SLES12-SP5-Updates' is up to date.

Repository 'SLE-SDK12-SP5-Pool' is up to date.

Repository 'SLE-SDK12-SP5-Updates' is up to date.

Repository 'SLE-Module-Toolchain12-Pool' is up to date.

Repository 'SLE-Module-Toolchain12-Updates' is up to date.

Repository 'SLE-Module-Web-Scripting12-Pool' is up to date.

Repository 'SLE-Module-Web-Scripting12-Updates' is up to date.

Retrieving repository 'Backport builds of Python Modules (SLE_12_SP5)' metadata [..

New repository or package signing key received:

Repository: Backport builds of Python Modules (SLE_12_SP5)

Key Name: devel:languages:python OBS Project <devel:languages:python@build.opensuse.org>

Key Fingerprint: 172CCD6B 0CF49B73 88F1C611 27163A4E EDF0D733

Key Created: Wed 10 Apr 2019 05:54:44 PM UTC

Key Expires: Fri 18 Jun 2021 05:54:44 PM UTC

Rpm Name: gpg-pubkey-edf0d733-5cae2de4

Cannot read input: bad stream or EOF.

If you run zypper without a terminal, use '--non-interactive' global

option to make zypper use default answers to prompts.

Do you want to reject the key, trust temporarily, or trust always? [r/t/a/? shows all options] (r): error]

Repository 'Backport builds of Python Modules (SLE_12_SP5)' is invalid.

[devel_languages_python_backports|https://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_12_SP5/] Valid metadata not found at specified URL

Please check if the URIs defined for this repository are pointing to a valid repository.

Skipping repository 'Backport builds of Python Modules (SLE_12_SP5)' because of the above error.

Repository 'openSUSE Factory Backports for SLE 12 (standard)' is up to date.

Some of the repositories have not been refreshed because of an error.

Installing R-3.6.3-1-1.x86_64.rpm...

Refreshing service 'Advanced_Systems_Management_Module_x86_64'.

Refreshing service 'Containers_Module_x86_64'.

Refreshing service 'HPC_Module_x86_64'.

Refreshing service 'Legacy_Module_x86_64'.

Refreshing service 'Public_Cloud_Module_x86_64'.

Refreshing service 'SUSE_Linux_Enterprise_Server_x86_64'.

Refreshing service 'SUSE_Linux_Enterprise_Software_Development_Kit_x86_64'.

Refreshing service 'Toolchain_Module_x86_64'.

Refreshing service 'Web_and_Scripting_Module_x86_64'.

Automatically trusting the following key:

Repository: Backport builds of Python Modules (SLE_12_SP5)

Key Name: devel:languages:python OBS Project <devel:languages:python@build.opensuse.org>

Key Fingerprint: 172CCD6B 0CF49B73 88F1C611 27163A4E EDF0D733

Key Created: Wed 10 Apr 2019 05:54:44 PM UTC

Key Expires: Fri 18 Jun 2021 05:54:44 PM UTC

Rpm Name: gpg-pubkey-edf0d733-5cae2de4

Building repository 'Backport builds of Python Modules (SLE_12_SP5)' cache [....done]

Loading repository data...

Reading installed packages...

Resolving package dependencies...

Problem: nothing provides libopenblas_pthreads-devel = 0.3.12 needed by openblas-devel-0.3.12-2.3.x86_64

Solution 1: do not install R-3.6.3-1-1.x86_64

Solution 2: break openblas-devel-0.3.12-2.3.x86_64 by ignoring some of its dependencies

Choose from above solutions by number or cancel [1/2/c] (c): c 
```
</details>

**Comment by glin** [comment]
Sorry for the late response. @kevinhankens and I ran into this a while back and found that the python-backports repo had published a broken `openblas-devel` package. Unfortunately, it still doesn't seem to be fixed, so it might be time to switch repos or drop OpenBLAS for openSUSE 42 / SLES 12 completely.

As a workaround, you can use one of the alternative repos from https://github.com/rstudio/r-builds/pull/50. The other repos only officially support SLES 12 SP4, not SP5, but they do work on SP5:
```sh
zypper --gpg-auto-import-keys addrepo https://download.opensuse.org/repositories/Cloud:/Platform:/Stack-SLE:/packages/SLE_12_SP4/Cloud:Platform:Stack-SLE:packages.repo
```

**Comment by christierney** [comment]
trying the alternative repo gives me:

```
Repository 'Additional packages required by SLE Stack (SLE_12_SP4)' is invalid.
[Cloud_Platform_Stack-SLE_packages|https://download.opensuse.org/repositories/Cloud:/Platform:/Stack-SLE:/packages/SLE_12_SP4/] Valid metadata not found at specified URL
Please check if the URIs defined for this repository are pointing to a valid repository.
Skipping repository 'Additional packages required by SLE Stack (SLE_12_SP4)' because of the above error.
```

**Comment by glin** [comment]
R binaries with the fix from #83 should now be available. To get the fix, remove the workaround repo and use the same instructions as before to install R on SLES 12.
